app/models/recipe.rb

```diff
@@ -0,0 +1,6 @@
+class Recipe < ApplicationRecord
+  has_many :recipe_foods
+  has_many :foods, through: :recipe_foods
```

Comment:
Consider adding missing attributes such as Cooking & Preparation time, description and public.

Reply:
@amentetteh notted the requested changes will be included in the future commits.